app/assets/javascripts/app/views.js

```diff
@@ -65,7 +65,8 @@ app.views.Base = Backbone.View.extend({
     var self = this;
     _.each(this.subviews, function(property, selector){
       var view = _.isFunction(self[property]) ? self[property]() : self[property];
-      if(view) {
+      if (view && self.$(selector).length > 0) {
+        self.$(selector).first().empty();
```

Comment:
@svbergerem, I managed to write a test that shows the issue that is solved by this line and that fails without it. I shaped it as a separate commit within the PR: https://github.com/diaspora/diaspora/pull/6864/commits/b532754c87581001827161366acdeb99c2bcdaf9